### ISSUE TITLE
Extend the loader with default module behavior

### ISSUE THREAD

**Issue by padcom** [itsmepetrov/classnames-loader#4]
In the current version the only way you can interact with the loaded stylesheet is basically to call it as a function. The default way, however, is to get the styles as fields.

It'd be great if we could merge the two approaches into one so that migration from regular styles to classNames is more seamless.

See the following for example implementation and usage:
https://github.com/padcom/react-example-02/blob/master/src/build/classnames-loader.js
https://github.com/padcom/react-example-02/blob/master/src/main/components/Input.js

So basically the imported variable is both an object and a function you can call.

**Comment by itsmepetrov** [comment]
Hi @padcom, I think it sounds good! 👍 
Could you please update readme (add and new example)?
Also maybe add an option to disable this feature?

**Comment by padcom** [comment]
Hi,

sorry for the late reply. Sure, I'll add that to the docs - makes sense. I
am however not that far into the loader API to make it configurable and
quite frankly I don't think it is necessary since both options are not
mutually exclusive at any point.

Cheers,
M.

2016-07-02 0:25 GMT+02:00 Anton Petrov notifications@github.com:

**Comment by padcom** [comment]
Done :)

2016-07-10 9:59 GMT+02:00 Matthias Hryniszak padcom@gmail.com:

**Comment by itsmepetrov** [comment]
@padcom, thanks for your great work!
